deploy/helm/CHANGELOG.md

```diff
@@ -465,7 +482,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
-- Scrape kube_service_* and kube_endpoint_* metrics
+- Scrape kube*service*_ and kube*endpoint*_ metrics
```

Comment:
@gantrior You might want to adjust your IDE. You keep refactoring all markdown files you touch. And the automatic refactoring can cause errors like this.

Reply:
damn you are right

Reply:
I reverted those changes